watsononlinestore/watson_online_store.py

```diff
@@ -318,9 +333,10 @@ def run(self):
                     print("message:\n %s\n channel:\n %s\n" %
                           (message, channel))
                 if message and channel:
-                    get_input = self.handle_message(message, channel)
+                    sender = SlackSender(self.slack_client, channel)
```

Comment:
A few lines up the code allows channel to be set. So for now it needs to be in the loop. There probably should be some conditionals for channel and user changes (to only do stuff first time for example).